pennylane/qaoa/cost.py

```diff
@@ -143,7 +151,7 @@ def edge_driver(graph, reward):
 
     allowed = ["00", "01", "10", "11"]
 
-    if not all([e in allowed for e in reward]):
+    if not all(e in allowed for e in reward):
```

Comment:
🎉

Reply:
that was from trying to get codefactor to pass.  I wonder if we can periodically trigger it on **all** of PL, instead of just what's nearby a recent change.